src/components/Popover/index.tsx

```diff
@@ -98,6 +101,8 @@ export default function Popover({
   const [referenceElement, setReferenceElement] = useState<HTMLDivElement | null>(null)
   const [popperElement, setPopperElement] = useState<HTMLDivElement | null>(null)
   const [arrowElement, setArrowElement] = useState<HTMLDivElement | null>(null)
+  // In default case where delayShowTimeout is 0 (no delay before showing) shouldShow initializes to true.
+  const [shouldShow, setShouldShow] = useState(Boolean(delayShowTimeout))
```

Comment:
Right now, it will initialize to `false`. I think you want:

```suggestion
  const [shouldShow, setShouldShow] = useState(show && delayShowTimeout === 0)
```

Reply:
oof yea thats right done

Reply:
done